tests/gnrc_tcp/Makefile

```diff
@@ -0,0 +1,30 @@
+include ../Makefile.tests_common
+
+# Basic Configuration
+BOARD ?= native
+
+# Shorten default TCP timeouts to speedup testing
+MSL_US ?= 1000000
+TIMEOUT_US ?= 3000000
+
+# Mark Boards with insufficient memory
+BOARD_INSUFFICIENT_MEMORY := airfy-beacon arduino-duemilanove arduino-mega2560 \
+                             arduino-uno calliope-mini chronos hifive1 mega-xplained microbit \
+                             msb-430 msb-430h nrf51dongle nrf6310 nucleo-f031k6 \
+                             nucleo-f042k6 nucleo-f303k8 nucleo-l031k6 nucleo-f030r8 \
+                             nucleo-f070rb nucleo-f072rb nucleo-f302r8 nucleo-f334r8 nucleo-l053r8 \
+                             sb-430 sb-430h stm32f0discovery telosb \
+                             wsn430-v1_3b wsn430-v1_4 yunjia-nrf51822 z1
```

Comment:
Please clear this list for now (or comment it out) so we can see which boards really don't fit.

Reply:
Done